src/components/tab-group/tab-group.spec.ts

```diff
@@ -0,0 +1,134 @@
+import {
+    it,
+    expect,
+    beforeEach,
+    inject,
+    describe
+} from '@angular/core/testing';
+import {TestComponentBuilder, ComponentFixture} from '@angular/compiler/testing';
+import {MD_TAB_GROUP_DIRECTIVES, MdTabGroup} from './tab-group';
+import {Component} from '@angular/core';
+import {By} from '@angular/platform-browser';
+
+describe('MdTabGroup', () => {
+  let builder: TestComponentBuilder;
+
+  beforeEach(inject([TestComponentBuilder], (tcb: TestComponentBuilder) => {
+    builder = tcb;
+  }));
+
+  it('should default to the first tab', (done: () => void) => {
+    builder.createAsync(TestApp).then(fixture => {
+      checkSelectedIndex(fixture, 1);
+      done();
+    });
+  });
+
+  it('should change selected index on click', (done: () => void) => {
+    return builder.createAsync(TestApp).then(fixture => {
+      let component = fixture.debugElement.componentInstance;
+      component.selectedIndex = 0;
+      checkSelectedIndex(fixture, 0);
+
+      // select the second tab
+      let tabLabel = fixture.debugElement.query(By.css('.md-tab-label:nth-of-type(2)'));
```

Comment:
It will probably be much cleaner to do the `createAsync` in a beforeEach, and then capture all of the tab labels in an array and access them directly in each test. I'm doing in in my in-progress rewrite of the radio button tests, I can show you if you're curious.